vsintegration/src/FSharp.Editor/LanguageService/FSharpProjectOptionsManager.fs

```diff
@@ -255,29 +250,37 @@ type private FSharpProjectOptionsReactor (workspace: VisualStudioWorkspaceImpl,
             while true do
                 match! agent.Receive() with
                 | FSharpProjectOptionsMessage.TryGetOptionsByDocument(document, reply, cancellationToken) ->
-                    try
-                        // For now, disallow miscellaneous workspace since we are using the hacky F# miscellaneous files project.
-                        if document.Project.Solution.Workspace.Kind = WorkspaceKind.MiscellaneousFiles then
+                    if cancellationToken.IsCancellationRequested then
```

Comment:
It seems as if cancellationToken is no longer passed in as an argument to tryComputeOptions so for my education where does this come from?

Reply:
oh i see now.

Reply:
We need to pass the CT to calls to grab textversion and text. I should not have removed it.